main.go

```diff
@@ -29,7 +29,7 @@ import (
 
 var (
 	// Version holds the current Gitea version
-	Version = "1.9.0-dev"
+	Version = "develop"
```

Comment:
Sorry, but we might just consider...
```suggestion
	Version = "development"
```

Reply:
I added it to the choices. We can wait a little more to choose.